src/harness/testcases/WINNF_FT_S_EPR_testcase.py

```diff
@@ -31,7 +33,11 @@ def setUp(self):
     self._sas_admin.Reset()
 
   def tearDown(self):
-    pass
+    logging.info('Stopping all running servers, if any')
```

Comment:
Maybe it makes sense to add this as a method to SasTestcase so that the code does not diverge later?